index.scss

```diff
@@ -138,38 +138,73 @@ $breakpoints-ie8: false!default;
   @return $value;
 }
 
+@function _set-max-breakpoint($max-breakpoint) {
+  @if $max-breakpoint != false {
+    @if unit($max-breakpoint) == px {
+      $max-breakpoint: $max-breakpoint - 1;
+    } @else {
+      $max-breakpoint: $max-breakpoint;
+    }
+  }
+
+  @return $max-breakpoint;
+}
+
 
 @mixin breakpoint($string) {
```

Comment:
this is the compiled version. will make it automated https://github.com/RazvanDH/rs-golden-ratio-grid/issues/4